app/src/main/java/co/tinode/tindroid/MessageActivity.java

```diff
@@ -202,6 +209,14 @@ public void onResume() {
                 showFragment(FRAGMENT_FILE_PREVIEW, args, true);
             }
         }
+
+        final SharedPreferences pref = PreferenceManager.getDefaultSharedPreferences(this);
+
+        mSendReadReceipts = pref.getBoolean(UiUtils.PREF_READ_RCPT, true);
+        Log.i(TAG, "Topic '" + topicName + "': sendReadReceipts = " + mSendReadReceipts);
```

Comment:
This looks like debugging output and completely unnecessary once debugged.

Reply:
Indeed. Removed.